packages/teko/src/Tpetra/Teko_InterlacedTpetra.cpp

```diff
@@ -243,12 +243,48 @@ RCP<Tpetra::CrsMatrix<ST,LO,GO,NT> > buildSubBlock(int i,int j,const RCP<const T
    LO numMyRows = rowMap.getNodeNumElements();
    LO maxNumEntries = A->getGlobalMaxNumRowEntries();
 
-   RCP<Tpetra::CrsMatrix<ST,LO,GO,NT> > mat = Tpetra::createCrsMatrix<ST,LO,GO,NT>(rcpFromRef(rowMap),maxNumEntries);
-
    // for extraction
    std::vector<GO> indices(maxNumEntries);
    std::vector<ST> values(maxNumEntries);
 
+   // for counting row sizes
+   std::vector<size_t> numEntriesPerRow(numMyRows,0);
```

Comment:
You don't actually need the zero here; `std::vector` invokes the default constructor of `size_t` for all the entries.